### DISCUSSION TITLE
2024/11/15/stata%E6%A8%A1%E6%8B%9F%E4%B8%AD%E5%BF%83%E6%9E%81%E9%99%90%E5%AE%9A%E7%90%86/

### DISCUSSION THREAD

**Discussion by giscus** [MrWillCom/MrWillCom.github.io#12] [Announcements]
# 2024/11/15/stata%E6%A8%A1%E6%8B%9F%E4%B8%AD%E5%BF%83%E6%9E%81%E9%99%90%E5%AE%9A%E7%90%86/

/* 设置文本为衬线体 */   .serif-text {     font-family: 'Georgia', 'Times New Roman', serif;   }   关于中心极限定理的模拟 do文件如下：   clear set seed 101   // 让我给试出个好看的图 set obs 50000  // 设置 50000 个观测值，用于存储均值  * 预先生成 gr

https://daseinzc.github.io/2024/11/15/stata%E6%A8%A1%E6%8B%9F%E4%B8%AD%E5%BF%83%E6%9E%81%E9%99%90%E5%AE%9A%E7%90%86/

**Comment by daseinzc** [comment]
欢迎评论！嘻嘻

**Reply by MrWillCom** [reply]
@daseinzc 你好，很高兴看见你为博客配置了评论系统，但是你的评论跑到我仓库的讨论区了，你的评论配置可能存在问题。请记得修复，我过段时间会关闭该讨论。谢谢！
